src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -277,9 +299,10 @@
         <!-- Install command -->
         <div class="section">
           <div class="section-header">
-            <span class="stepcircle" hidden$="[[!isGenericCloudProvider]]">1</span>
-            <span class="stepcircle" hidden$="[[isGenericCloudProvider]]">2</span>
-            <div class="instructions">[[localize('manual-server-install-run')]]</div>
+            <span class="stepcircle">{{installScriptStepNumber}}</span>
```

Comment:
nit: one-way bindings (i.e. `[[installScriptStepNumber]]`) should be sufficient here and below.

Reply:
Done in d782227